api/v1alpha1/blockdevice_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The OpenEBS Authors
+Copyright 2021.
```

Comment:
[This](https://github.com/openebs/node-disk-manager/blob/master/build/custom-boilerplate.go.txt) is the license header that should be used. You can edit the file to 2021.